pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
+line-length = 67
```

Comment:
This line size was defined in this way so that the examples in the book fit inside the text box without losing formatting